src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int START_DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOURS_OF_WORK_INDEX = 2;
+    private static final int SALARY_PER_HOUR_INDEX = 3;
+    private static final int DAY_TO_PLUS = 1;
+    private static final String DATE_FORMAT_PATTERN = "dd.MM.yyyy";
+    private static final DateTimeFormatter dateTimeFormatter
+            = DateTimeFormatter.ofPattern(DATE_FORMAT_PATTERN);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder(
+                "Report for period " + dateFrom + " - " + dateTo + "\n");
```

Comment:
be attentive to the previous comments
let's use append() instead of +
```suggestion
        StringBuilder stringBuilder = new StringBuilder("Report for period ")
            .append(dateFrom).append(" - ").append(dateTo).append("\n");
```